neurokit2/ecg/ecg_findpeaks.py

```diff
@@ -1051,6 +1053,80 @@ def _ecg_findpeaks_rodrigues(signal, sampling_rate=1000, **kwargs):
     return rpeaks
 
 
+# =============================================================================
+# Visibility graph transformation - by Koka and Muma (2022)
+# =============================================================================
+def _ecg_findpeaks_vgraph(signal, sampling_rate=1000, lowcut=3, order=2, **kwargs):
+    """R-Peak Detector Using Visibility Graphs by Taulant Koka and Michael Muma (2022).
+
+    References
+    ----------
+    - T. Koka and M. Muma (2022), Fast and Sample Accurate R-Peak Detection for Noisy ECG Using
+      Visibility Graphs. In: 2022 44th Annual International Conference of the IEEE Engineering
+      in Medicine & Biology Society (EMBC). Uses the Pan and Tompkins thresholding.
+
+    """
+    # Try loading ts2vg
+    try:
+        from ts2vg import NaturalVG
```

Comment:
just for clarity, I prefer to have the name of the package as a prefix to its function calls, it helps me to remind from where each function comes

```suggestion
        import ts2vg
```